widgets/cloud-table/cloud-table.tsx

```
@@ -0,0 +1,305 @@
+    const value = cloud[accessorKey as keyof Cloud];
+
+    switch (accessorKey) {
+      case "provider":
+        return (
+          <Badge variant="outline" className="font-medium">
+            {value as string}
+          </Badge>
+        );
+
+      case "cloudGroupName":
+        const groups = value as string[] | undefined;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)